sdk/spring/azure-spring-boot-test-aad/README.md

```diff
@@ -2,7 +2,78 @@
+1. Select Conditional Access button.![Select Button](docs/image-conditional-access-button.png)
+1. Then create a new policy.![Create Policy](docs/image-conditional-access-new-policy.png)
+1. We need to prepare a user or a group. As below, we need to configure users or groups to follow this policy.![Add User](docs/image-conditional-access-add-user.png)
+1. As before, we need to configure one or more applications to follow this policy. In our case, we need to configure the ***webapiB*** application.![Add Application](docs/image-conditional-access-add-application.png)
+1. In our case, we use [Multi-Factor Authentication] as Conditional Access Policy. So select `Require multi-factor authentication` in ***Grant***.![API Policy](docs/image-conditional-access-add-MFA.png)
+1. Finally, enable it.(If can’t create a policy, maybe you need to close [Access management for Azure resources].)![Enable Conditional Access](docs/image-conditional-access-enable.png)
+
+##### Run AADConditionalAccessIT
+
+This case is difficult to run automatically, we need to execute it manually. 
+Here are the steps to start this case.
+
+ - Prepare application and Conditional Access Policy as above.
+ - Prepare environment variables for `ConditionalAccessPolicyTestWebApiB`(***com.azure.test.aad.conditional.access.webapi.b***) as below. The environment variables are the parameters of our ***webapiB*** application and application server port Will start at 8883.
+
+| webapiB application requires environment variable name | Actual Value | 
+| ---: | :----: | 
+|   CONDITIONAL_ACCESS_POLICY_TEST_WEB_API_B_CLIENT_ID  | your webapiB application id |
+
+ - Same as `ConditionalAccessPolicyTestWebApiB`, The environment variables of `ConditionalAccessPolicyTestWebApiA`(***com.azure.test.aad.webapi.conditional.access.webapi.a***) is the parameters of our ***webapiA*** application and  application server port Will start at 8882.
+
+| webapiA application requires environment variable name | Actual Value | 
+| ---: | :----: | 
+|   CONDITIONAL_ACCESS_POLICY_TEST_WEA_API_A_CLIENT_ID  | your webapiA application id |
+|   CONDITIONAL_ACCESS_POLICY_TEST_WEB_API_A_CLIENT_SECRET  | your webapiA application secret |
+|   CONDITIONAL_ACCESS_POLICY_TEST_WEA_API_B_CLIENT_ID  | your webapiB application id |
+|   AAD_TENANT_ID_1  | your tenant id |
+
+ - At last, Prepare the environment variables of `AADConditionalAccessIT` through the parameters of ***webapp*** application.
+
+| webapp application requires environment variable name | Actual Value | 
+| ---: | :----: | 
+|   CONDITIONAL_ACCESS_POLICY_TEST_WEB_API_A_CLIENT_ID  | your webapiA application id |
+|   AAD_SINGLE_TENANT_CLIENT_ID  | your webapp application id |
+|   AAD_SINGLE_TENANT_CLIENT_SECRET  | your webapp application secret |
+|   AAD_TENANT_ID_1  | your tenant id |
+|   AZURE_CLOUD_TYPE  | your azure subscription type |
+
+ - Start the `conditionalAccessTest()` method.
```

Comment:
nit: We should start `ConditionalAccessPolicyTestWebApiB` and `ConditionalAccessPolicyTestWebApiA` first.